eva/catalog/models/base_model.py

```diff
@@ -109,8 +109,9 @@ def init_db():
     if not database_exists(engine.url):
         logger.info("Database does not exist, creating database.")
         create_database(engine.url)
-        logger.info("Creating tables")
-        BaseModel.metadata.create_all()
+
```

Comment:
We may want to add a comment clarifying that some database systems like Postgres have a default database?